org.gnome.EasyTAG.desktop

```diff
@@ -0,0 +1,94 @@
+[Desktop Entry]
```

Comment:
Would be nice to submit changes to this file upstream.

Reply:
The master branch of upstream actually already has the required change, which is the icon name, however it's in the form before the translations are added so I can't just download it straight out of there (unless there is actually a straightforward way to just generate it from the .desktop.in file and the localisations, I'm quite new at all of this).

Reply:
You should use rename-icon, rename-desktop-file & so on. See how other apps do that here on Flathub.

Reply:
Actually the reason that I was copying the icons is that the About dialog in particular still refers to the icon by its old name, but apparently it's not even accessible in Wayland, and upstream master has already fixed the icon names, so I suppose I can drop that and switch to rename-icon.

Reply:
After commit cf062c5 rename-icon, rename-desktop-file and rename-appdata-file are now used. I have also submitted a PR to upstream [here](https://gitlab.gnome.org/GNOME/easytag/-/merge_requests/9).